org.kde.labplot2.json

```diff
@@ -79,10 +97,15 @@
             ],
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://github.com/qt/qtmqtt.git",
-                    "commit": "cade213f562049400da81e60762400df68d7c56b",
-                    "tag": "v5.15.2"
+                    "type": "archive",
+                    "url": "https://github.com/qt/qtmqtt/archive/refs/tags/v6.2.3.tar.gz",
```

Comment:
the version is related to the Qt version. 6.2.3 is not working. We need 5.15.2.

Reply:
Since the application needs specific version of Qt, is it fine if I just leave out Qt from auto updating?

Reply:
sure.

Reply:
Updated